src/mui/field/SingleFieldList.js

```diff
@@ -0,0 +1,30 @@
+import React, { PropTypes } from 'React';
```

Comment:
Maybe it makes sense to move it to `list/` instead of `field/`, so that it's aside `Datagrid`?